docs/docs/fargate-tutorials/prisma.md

```diff
@@ -134,126 +140,214 @@ node index.js
- create | public_repository   |      2 | quickstart-repository
- create | cluster             |      2 | 2
- create | task_definition     |      2 | 2
- create | service             |      2 | 2
- create | listener            |      2 | 2
- create | load_balancer       |      2 | 2
- create | target_group        |      2 | 2
- create | security_group      |      5 | 5
- create | security_group_rule |      3 | 3
- create | security_group_rule |      4 | 4
- create | role                |        | ecsTaskExecRole
-```
-
-## Login, build and push your code to the container registry
-
-1. Grab your new `ECR URI` from the hosted DB
-```bash
-QUICKSTART_ECR_URI=$(psql -At 'postgres://d0va6ywg:nfdDh#EP4CyzveFr@db.iasql.com/_4b2bb09a59a411e4' -c "
-SELECT repository_uri
-FROM repository
-WHERE repository_name = '<project-name>-repository';")
-```
-
-2. Login to AWS ECR using the AWS CLI. Run the following command and use the correct `<ECR-URI>` and AWS `<profile>`
-
-```bash
-aws ecr get-login-password --region ${AWS_REGION} --profile <profile> | docker login --username AWS --password-stdin ${QUICKSTART_ECR_URI}
-```
-
-:::caution
-
-Make sure the [CLI is configured with the same credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html), via environment variables or `~/.aws/credentials`, as the ones provided to IaSQL or this will fail.
-
-:::
-
-3. Build your image locally
-
-```bash
-docker build -t <project-name>-repository app
-```
+  const cbData = {
+    role_name: cbRole,
+    assume_role_policy_document: assumeServicePolicy,
+    attached_policies_arns: [codebuildPolicyArn, cloudwatchLogsArn, pushEcrPolicyArn]
+  }
+  await prisma.role.upsert({
+    where: { role_name: cbRole },
+    create: cbData,
+    update: cbData,
+  });
 
-4. Tag your image
+  const buildSpec = `
+    version: 0.2
+
+    phases:
+      pre_build:
+        commands:
+          - echo Logging in to Amazon ECR...
+          - aws ecr get-login-password --region ${region} | docker login --username AWS --password-stdin ${repoUri}
+      build:
+        commands:
+          - echo Building the Docker image...
+          - docker build -t ${appName}-repository examples/ecs-fargate/prisma/app
+          - docker tag ${appName}-repository:latest ${repoUri}:latest
+      post_build:
+        commands:
+          - echo Pushing the Docker image...
+          - docker push ${repoUri}:latest
+  `;
+
+  const pjData = {
+    project_name: appName,
+    source_type: 'GITHUB',
+    service_role_name: cbRole,
+    source_location: ghUrl,
+    build_spec: buildSpec,
+  };
+  await prisma.codebuild_project.upsert({
+    where: { project_name: appName},
+    create: pjData,
+    update: pjData,
+  });
+  await prisma.codebuild_build_import.create({
+    data: {
+      project_name: appName,
+    }
+  });
 
-```bash
-docker tag <project-name>-repository:latest ${QUICKSTART_ECR_URI}:latest
+  console.dir(await prisma.$queryRaw`SELECT * from iasql_apply();`)
```

Comment:
is this debug message intended to be here?

Reply:
yes, this is to display the output of the `apply`